code/__DEFINES/dcs/signals.dm

```diff
@@ -28,6 +28,7 @@
 #define COMSIG_GLOB_MOB_LOGIN "!mob_login"
 #define COMSIG_GLOB_MOB_LOGOUT "!mob_logout"
 #define COMSIG_GLOB_MOB_DEATH "!mob_death"
+	#define DEATH_STOPPED (1<<0)
```

Comment:
```suggestion
#define DEATH_STOPPED (1<<0)
```

Reply:
Nop, that's standard for bit flag sent by signals

Reply:
^^